tests/MessageTest.php

```diff
@@ -101,6 +102,7 @@ public function testDelete()
 
         $message = $this->mailbox->getMessage(3);
         $message->delete();
+        $this->mailbox->expunge();
```

Comment:
Trivial bugfix. [Manual](https://secure.php.net/manual/en/function.imap-delete.php) says:

> Marks messages listed in msg_number for deletion. Messages marked for deletion will stay in the mailbox until either imap_expunge() is called or imap_close() is called with the optional parameter CL_EXPUNGE.

Reply:
If we only fix this here, this may not be transparent to users. Should we also add a doc comment to `Message::delete()`?

Reply:
So, since `delete` doesn't delete the message, I suggest:

1. Rename `delete` method to `markAsDelete`
1. Add to the doc that `mailbox::expunge` must be called for a complete delete

Reminder: https://github.com/ddeboer/imap/issues/175